integration/commands_administration_test.go

```diff
@@ -919,6 +922,43 @@ func TestCommandsAdministrationServerStatusMetrics(t *testing.T) {
 	}
 }
 
+func TestCommandsAdministrationServerStatusStressPostgres(t *testing.T) {
+	setup.SkipForPostgresWithReason(t, "https://github.com/FerretDB/FerretDB/issues/1317")
+	setup.SkipForTigrisWithReason(t, "Tigris needs a schema for collection creation")
+
+	t.Parallel()
+
+	ctx, collection := setup.Setup(t) // no providers there, we will create collections concurrently
+	client := collection.Database().Client()
+
+	collNum := 10
+
+	var wg sync.WaitGroup
+	for i := 0; i < collNum; i++ {
+		wg.Add(1)
+
+		go func(i int) {
+			dbName := fmt.Sprintf("%s_stress_%d", collection.Database().Name(), i)
+			db := client.Database(dbName)
+			_ = db.Drop(ctx)
+
+			collName := fmt.Sprintf("stress_%d", i)
+			_ = db.CreateCollection(ctx, collName)
+			_ = db.Drop(ctx)
```

Comment:
We expect no errors from dropping DB and collections once schema status is fixed right?

Reply:
I think it's not important here, at least for now. My idea was to do some weird things to have a stable way to reproduce errors from running `serverStatus` command, and this approach gives 100% reproducible errors on my machine.

Reply:
Sounds good :+1: